Practice/Practice1/Practice/Program.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Practice
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            int a = 7;
+            int k = 4;
+            double b = 5.3;
+            char z = 'A';
```

Comment:
please write appropriate name of variables

Reply:
Noted